### PR TITLE
Get payout tiers for affiliates.

### PR DESCRIPTION
**Pull request by shekhardokania**

### CODE REVIEW

**Comment by psychocandy** [comment]
Can you rebase/solve conflicts?

**Comment by shekhardokania** [comment]
Done.

File: spec/lib/application_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe HasOffersV3::Application do
+  let(:url)  { Regexp.new api_url('Application') }
```
Comment (psychocandy) [inline spec/lib/application_spec.rb]: why do you need `Regexp` here?

Comment (shekhardokania) [inline spec/lib/application_spec.rb]: because it's a get request and the query string is also appended to the url.

Comment (shekhardokania) [inline spec/lib/application_spec.rb]: due to which mocking the request without regexp is not possible.

File: spec/lib/application_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  describe '.get_payout_tiers' do
+    it 'should make a proper request call' do
+      stub_call :get
+      response = HasOffersV3::Application.get_payout_tiers
```
Comment (psychocandy) [inline spec/lib/application_spec.rb]: I prefer:
```ruby
subject { HasOffersV3::Advertiser }
# code code code
subject.get_payout_tiers
```

Comment (shekhardokania) [inline spec/lib/application_spec.rb]: yeah but here only one example was there, so took the easy path. :)

File: spec/lib/application_spec.rb (hunk 1 above)
Comment (driv3r) [inline spec/lib/application_spec.rb]: yes, it the same as in Report, so it should be fine

**Comment by driv3r** [comment]
:+1:

**Pull request merged**